### DISCUSSION TITLE
"For you" shows release notes for SSO org without authentication

### DISCUSSION THREAD

**Discussion by kassner** [community/community#53713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I use the same GitHub account for work and personal projects. For work stuff, I have to use SSO to be able to access anything from the organization, and that SSO is only available on my work computer. However, I see releases from private repositories from that organization on my "For you" page on my personal computer, where I haven't SSO'd into the organization.

**Comment by lRaulMN7** [comment]
+1, this is kinda annoying :( I'd be amazing to have a toggle to filter between personal feed and SSO feed.

<img width="540" alt="Screenshot 2024-06-02 at 12 25 39" src="https://github.com/community/community/assets/39067731/16cf9f3d-8ea2-4fa4-b69b-e172652c7886">
